src/www/status_dhcp_leases.php

```diff
@@ -246,15 +246,19 @@ function remove_duplicate($array, $field)
 
     $order = ( $_GET['order'] ) ? $_GET['order'] : 'ip';
 
-    usort($leases,
-        function ($a, $b) use ($order) {
+    if ($order === 'ip') {
```

Comment:
this looks a bit convoluted. If we pass $order top the usort function we only need one which does inline if/else on the function to use?

Reply:
Not sure whether I understand your idea right. In commit 2adf500 I put the $order condition inside the usort function and it looked quite nice. Then I realised that we can eliminate this condition being checked multiple times while sorting by means of splitting the single usort function into two ones: a simple one for sorting by IP (no need to even have $order inside the function namespace) and an extra one for sorting by arbitrary field chosen by the user. The latter approach is what I implemented in commit a012f89.

Reply:
This inflates size about a factor of four compared to when the code was when this lease cleanup started. I understand that adding code is necessary to improve functionality but as a leaf page we don't need to prematurely optimise it.